config/game_options.txt

```diff
@@ -18,9 +18,7 @@ REVIVAL_BRAIN_LIFE -1
 ## Comment this out if you want OOC to be automatically disabled during the round, it will be enabled during the lobby and after the round end results.
 OOC_DURING_ROUND
 
-## LOOC
-## Comment this out to disable LOOC
-LOOC_ENABLED
+## LOOC was removed by nsv13
```

Comment:
```suggestion
## LOOC is now an in-game toggle
```

Reply:
whoops, i can see how that can be misconstrued